Added MAxim as co-author (#1562)
OWASP/owasp-mstg: Document/0x02-Frontispiece.md

Jeroen is the mobile security lead at NVISO where he is responsible for quality assurance on mobile security projects and for R&D on all things mobile. He worked as a Flash developer during high school and college, but switched to a career in cybersecurity once he graduated and now has more than 5 years of experience in mobile security. He loves sharing his knowledge with other people, as is demonstrated by his many talks & trainings at colleges, universities, clients and conferences.

#### Maxim Gal

> Maxim has a Bachelor in Software Engineering and was working as an iOS Engineer since 2011. His working experience covers startup companies to the financial industry. In the last two years he shifted to the role of a Mobile Application Security Engineer and was contributing to both projects, the MSTG and MASVS. Maxim works currently at Sberbank and is always looking for the next challenge.

Reviewer: Lowercased some words as they should be; added "from", please check if that's the intended meaning; 
```suggestion
Maxim has a bachelor in software engineering and was working as an iOS engineer since 2011. His working experience covers from startup companies to the financial industry. In the last two years he shifted to the role of a mobile application security engineer and was contributing to the MSTG and the MASVS. Maxim works currently at Sberbank and is always looking for the next challenge.
```

Reviewer 2: We should write it like this: Bachelor of Software Engineering (B.SE.)

See:
https://en.wikipedia.org/wiki/Bachelor_of_Software_Engineering
